src/main/resources/view/ItemListCard.fxml

```diff
@@ -12,7 +12,7 @@
 <?import javafx.scene.text.Text?>
 <?import javafx.scene.text.TextFlow?>
 
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Is this change to /11.0.1 automated by your intellij system or you changed it manually?

Reply:
oh didn't realise that happened, must have been automatic